spyder/widgets/tests/test_collectioneditor.py

```diff
@@ -265,6 +265,25 @@ def test_filter_rows(qtbot):
     assert editor.model.rowCount() == 0
 
 
+def test_remote_make_data_function():
+    """
+    Test that the function returned by make_data_function() ...
```

Comment:
```suggestion
    Test that the function returned by make_data_function() is the expected
    one.
```

I'm not sure if this is really what this docstring should say. If not, please edit my suggestion before applying it.